### DISCUSSION TITLE
求激活码

### DISCUSSION THREAD

**Discussion by maria202301** [LFTEC/LearningFucker#33] [General]
------------------------------License------------------------------
d56yltS/oeDzSE9OcXJjDKhl5f1H7ebMXYJdAHwSLsdJRjTkjV+DuLbxsRPtfa2y+7r
ZErngSXXoURkUjEPSwZzveUYphrK3oLSKMUcFAAaG8wtBtK+rcjEYFqfu8bfzh42vH6
UtaaMwVOGJQ/YzK7q5HwXma9RL9Zre8OqJrQ44VdD2tFTpS7MSurZ7AxZLgfAT/e/iM
NSqJF5En6FX3mPao5tQB3COQRDk+JjKt1WSqOjFuey1EBzW+SNvSRYDq9mYOrbdgJXI
OF+esHVQBLmCH3TDMoGJXEzDemoe93B6qz7i9M2trXzvYWO2RuQfS7MUDI8irB10oN9
RyyF3xzevmRUl1AuF/OsxR3q0wm2UyCMH0I6g3Kw6l5ggUdHrrtb2CaipdYPZ8uQutH
cgRvzrlV1Hxx7h1alNAsf4aCDII2uhj+XHPWqrxSiLaBA7VmPb72BOsq/LiiYzTzfJt
5F/jrMQHaj54iUv/+Lvj8ZMrzHhg5Ulx/rqJAtuqlONfrEbzQHp7vtf+XUfJEnIRCdQ
0AM36Ys+3h/hxBf9KIkZKtAmuFAYkX9eXK0rPEBdIPVIaMLM39/PZ5OFEQo4YAUovWz
Mt56YiJsLga29zk3vHenzkWoQGAX4qD0k0bwhQuuAt/egN64IoMFG3VWKoPA4NbHJHQ
EShJYjYL/7kAWLRG7eVvv+VhBmqbilq21uBYJnoilJivck8EYg0xImE9yl00Z8xMR3K
WJ1qjAYpnn9iQZlDGVdUyxan0o8mHNNsYLBjz0Sfv3WkStju2TZbJEOljH1QK2TXSZS
zfO4K604oJJPoLyt4pKLt/LIyQZuzTKuuoC+XsDYQ8bJk13OMxNRnE7ROMDf9Lazzz1
h3VWZhvdN5lE9Fz1I/rlLTxw/fkHggPc8OYNCLuzP1QfP9zYTSVSCclLkhy0YvpkTPv
Xpj4vZqiP9NsFlJJAtihLpgKWD3ujC4DSgxEJap+5+6GGxp6iIVocmIDgCwfd31Nkgf
LExroS2wgKdblHvuMZ8Dvy1U4MjEjQg4GuQ+dhid8cHaRI7EEsRbBpU6jw/F9ZQDf6K
CvbfoZ203JhSfDCaKifccKpUQ9wpIOUGeBy4A0T4/JmKmdOMinHbnq4LBRdarYYrqYO
9yPd4Ov6ybau49lQtcHbMlO439DZwckpyIlAv5HghR3bbSwj9cg8RscKAwkmybEoQxl
tt/bVkFhEkV4ED9dE2Qh8BHo2lyQTspSGb5+SnftBCEW2DsyL5ClBqJi9BP++z04vj0
Pg/hX9eTEonBN5mYIcRXRXJqUXqJABYQYuhe6rUtPqwWOCHSXoYI5By9mBmjuUV/GmT
R0uxhsDVpRfuZ4V1+zYniDtO+w==
------------------------------License------------------------------

**Comment by LFTEC** [comment]
请使用正确的github账号激活
